modules/space/rendering/renderablefluxnodes.cpp

```diff
@@ -0,0 +1,1093 @@
+        "radiusPerspectiveEnabled",
+        "Include radius with cameraperspective",
+        "If false, then nodes closer to the sun will not be larger "
+        "regardless of distance to camera."
+    };
+    constexpr openspace::properties::Property::PropertyInfo 
+                                                        PerspectiveDistanceFactorInfo = {
+        "perspectiveDistanceFactor",
+        "Perspective Distance factor",
+        "This value decides how far away the camera must be to start "
+        "impacting the node size."
+    };
+    constexpr openspace::properties::Property::PropertyInfo MinNodeSizeInfo = {
+        "minNodeSize",
+        "Minimum node size",
+        "The minimum node size."
+    };
+    constexpr openspace::properties::Property::PropertyInfo MaxNodeSizeInfo = {
+        "maxNodeSize",
+        "Maximum node size",
+        "The minimum node size."
+    };
+    constexpr openspace::properties::Property::PropertyInfo AlwaysPulseInfo = {
+        "alwaysPulsate",
+        "Pulsate regardless of camera position",
+        "Always have nodes close to earth pulsate regardless of position."
+    };
+    constexpr openspace::properties::Property::PropertyInfo pulseEnabledInfo = {
+       "pulseEnabled",
+       "Nodes close to Earth pulsate",
+       "Toggles the pulse for nodes close to Earth."
+    };
+    constexpr openspace::properties::Property::PropertyInfo gaussianPulseEnabledInfo = {
+       "gaussianPulseEnabled",
+       "Nodes close to Earth pulsate with alpha by gaussian",
+       "Toggles the pulse with alpha by gaussian for nodes close to Earth."
+    };
+
+    float stringToFloat(const std::string input, const float backupValue = 0.f) {
+        float tmp;
+        try {
+            tmp = std::stof(input);
+        }
+        catch (const std::invalid_argument& ia) {
+            LWARNING(fmt::format(
+                "Invalid argument: {}. '{}' is NOT a valid number", ia.what(), input
+                ));
+            return backupValue;
+        }
+        return tmp;
+    }
+
+    struct [[codegen::Dictionary(RenderableFluxNodes)]] Parameters {
+        // path to source folder with the 3 binary files in it
+        std::string sourceFolder;
+        //
+        struct TransferFunctions {
+            std::string standard;
+            std::string flow;
+            std::string earth;
+            std::string cmr [[codegen::key("CMR")]];
+        };
+        // [[codegen::verbatim(ColorTablePathInfo.description)]]
+        TransferFunctions colorTablePaths;
+        // [[codegen::verbatim(LineWidthInfo.description)]]
+        //float lineWidth;
+        // [[codegen::verbatim(GoesEnergyBinsInfo.description)]]
+        std::optional<int> energyBin;
+    };
+#include "renderablefluxnodes_codegen.cpp"
+
+    // Changed everything from dvec3 to vec3
+    glm::vec3 sphericalToCartesianCoord(glm::vec3 position) {
```

Comment:
Since you are only reading from it, you can `const glm::vec3& position` the parameter instead